specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-01-01-preview/ApplicationGroups.json

```diff
@@ -0,0 +1,327 @@
+          "description": "Link to the next set of results. Not empty if Value contains an incomplete list of Authorization Rules"
+        }
+      },
+      "description": "The response from the List Application Groups operation."
+    },
+    "ApplicationGroup": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "updatedAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "Exact time the Application Group was updated"
+            },
+            "createdAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "Exact time the Application Group was created."
+            },
+            "isEnabled": {
+              "type": "boolean",
+              "description": "Determines if Application Group is allowed to create connection with namespace or not. Once the isEnabled is set to false, all the existing connections of application group gets dropped and no new connections will be allowed"
+            },
+            "appGroupIdentifier": {
+              "type": "string",
+              "description": "The Unique identifier for application group.Supports SAS(SASKeyName=KeyName) or AAD(AADAppID=Guid1)"
+            },
+            "appGroupName": {
+              "type": "string",
+              "description": "The Application Group Name"
+            },
+            "policies": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ThrottlingPolicy",
+                "description": "The Application Group Policy",
+                "x-ms-client-flatten": true
+              },
+              "description": "List of group policies that define the behavior of application group. The policies can support resource governance scenarios such as limiting ingress or egress traffic."
+            }
+          }
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system meta data relating to this resource.",
+          "$ref": "../../../common/v1/definitions.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v2/definitions.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "The Application Group object"
+    },
+    "ThrottlingPolicy": {
+      "type": "object",
+      "description": "Properties of the throttling policy",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The Name of this policy"
+        },
+        "rateLimitThreshold": {
+          "format": "int64",
+          "type": "integer",
+          "description": "The Threshold limit above which the application group will be throttled.Rate limit is always per second."
```

Comment:
do we have any upper/lower limit, if there is please add in the description.

Reply:
@yogeshgargmsft  Is there one? or is it limited it int size?